extensions/import/src/wizard/pages/fileConfigPage.ts

```diff
@@ -270,7 +270,9 @@ export class FileConfigPage extends ImportPage {
 		let connectionUri = await azdata.connection.getUriForConnection(this.model.server.connectionId);
 		let queryProvider = azdata.dataprotocol.getProvider<azdata.QueryProvider>(this.model.server.providerName, azdata.DataProviderType.QueryProvider);
 
-		let query = `SELECT name FROM sys.schemas`;
+		let databaseName = this.databaseDropdown.value.toString().replace(/]/g, ']]');
+
+		const query = this.databaseDropdown.value ? 'SELECT name FROM [${databaseName}].sys.schemas' : 'SELECT name FROM sys.schemas';
```

Comment:
You're checking for this.databaseDropdown.value to be truthy here but you've already used it in on line 273 - if it was undefined it would have failed there. 

Also I just looked at it myself and the dropdown value is a string | CategoryValue - and specifically in this instance we're populating it with CategoryValue objects. Did you test this yourself? Because I'm surprised it worked with the code you had. 

Something like this should work, but you should be testing this yourself too to make sure it works as expected. 

```
const escapedQuotedDb = this.databaseDropdown.value ? `[${(<azdata.CategoryValue>this.databaseDropdown.value).name.replace(/]/g, ']]')}` : '';
const query = `SELECT name FROM ${escapedQuotedDb}.sys.schemas`;
```